ApplicationCode/ProjectDataModel/RimDepthTrackPlot.cpp

```diff
@@ -380,6 +384,38 @@ void RimDepthTrackPlot::visibleDepthRange( double* minimumDepth, double* maximum
     *maximumDepth = m_maxVisibleDepth;
 }
 
+RiaDefines::DepthUnitType RimDepthTrackPlot::caseDepthUnit() const
+{
+    RimEclipseResultCase* thecase = dynamic_cast<RimEclipseResultCase*>( commonDataSource()->caseToApply() );
+    if ( thecase == nullptr )
+    {
+        // no suitable case found, look in the project to see if there is a eclipse case with units defined loaded
+        RimProject* p = RiaApplication::instance()->project();
+        for ( RimEclipseCase* aCase : p->activeOilField()->analysisModels()->cases() )
+        {
+            thecase = dynamic_cast<RimEclipseResultCase*>( aCase );
+            if ( thecase ) break;
+        }
```

Comment:
Maybe use  RimProject::eclipseCases() to get the RimEclipseCases? 

Maybe also make a findFirstEclipseCase to get rid of the "break;"?